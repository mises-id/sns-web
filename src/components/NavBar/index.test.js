/*
 * @Author: lmk
 * @Date: 2021-07-15 10:46:27
 * @LastEditTime: 2021-08-13 00:12:35
 * @LastEditors: lmk
 * @Description: CreateMisesIdtest
 */
import React from 'react';
import {shallow } from 'enzyme';
import Navbar from './';
describe('CreateMisesId', () => {
  it('should render correctly', () => {
    const component = shallow(<Navbar />)
    expect(component).toMatchSnapshot();
  });
});