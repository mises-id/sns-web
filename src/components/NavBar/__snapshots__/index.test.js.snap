// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateMisesId should render correctly 1`] = `
<NavBar
  left={
    <Icon
      onClick={[Function]}
      prefixCls="za-icon"
      size="sm"
      theme="default"
      type="arrow-left"
      viewBox="0 0 32 32"
    />
  }
  prefixCls="za-nav-bar"
/>
`;
